elasticapm/contrib/django/management/commands/elasticapm.py

```diff
@@ -163,6 +164,38 @@ def handle_check(self, command, **options):
             self.write(CONFIG_EXAMPLE)
         self.write('')
 
+        server_url = client.config.server_url
+        if server_url:
+            parsed_url = urlparse.urlparse(server_url)
+            if parsed_url.scheme.lower() in ('http', 'https', ):
+                # parse netloc, making sure people did not supply basic auth
+                if '@' in parsed_url.netloc:
+                    credentials, _, path = parsed_url.netloc.rpartition('@')
+                    passed = False
+                    self.write(
+                        'Configuration errors detected!', red, ending='\n\n'
+                    )
+                    if ':' in credentials:
+                        self.write('  * SERVER_URL cannot contain authentication '
+                                   'credentials', red, ending='\n')
+                    else:
+                        self.write('  * SERVER_URL contains an unexpected at-sign!'
+                                   ' This is usually used for basic authentication, '
+                                   'but the colon is left out', red, ending='\n')
+                else:
+                    self.write('Server url #lgtm', green)
```

Comment:
Can you use `SERVER_URL` here, same as in all the other messages? Also, not everybody might be familiar with "lgtm", so maybe just write "`SERVER_URL` looks good"

Reply:
Done. And added testcases